tests/unit/store/settings.spec.js

```diff
@@ -25,6 +25,12 @@ describe("store/settings", () => {
       .customer_account_ids;
     expect(array_of_customerids.length).toEqual(0);
   });
+  test("When imported from the dist file, Then the file_count and max_file_count is zero 0 as with no value assigned", () => {
```

Comment:
This test is not importing from the dist file.  The `store` variable is loaded from the regular files on line 17

Reply:
Corrected to `initialized`